modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/WebDataFile_conf.pm

```diff
@@ -0,0 +1,141 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+=pod
+
+=head1 NAME
+
+Bio::EnsEMBL::Production::Pipeline::PipeConfig::WebDataFile_conf
+
+=head1 DESCRIPTION
+
+This pipeline automate current manual processing of the 7 species (https://2020.ensembl.org/app/species-selector) currently available on 2020 website.
+
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::WebDataFile_conf;
+use strict;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+
+use Data::Dumper;
+
+sub default_options {
+    my $self = shift;
+    return {
+        %{$self->SUPER::default_options()},
+        species         => [],
+        division        => [],
+        antispecies     => [],
+        run_all         => 0, #always run every species
+        step             => [],  
+        genomeinfo_yml    => undef,
+        # Email Report subject
+        email_subject => $self->o('pipeline_name').'  pipeline has finished',
+        release => software_version()
```

Comment:
Neither of these parameters is used.

Reply:
i will add email_subject  in  analysis 'email_notification' as subject parameter  
release parameter currently not used but will be used in future

Reply:
The "release" variable is available via the parent conf, but it's called `ensembl_release`.

Reply:
sure i will make use of this variable